builtin/v16/evm/evm_state.go

```diff
@@ -9,6 +9,16 @@ import (
 	"github.com/filecoin-project/go-state-types/builtin/v16/util/adt"
 )
 
+type TransientDataLifespan struct {
+	Origin abi.ActorID
+	Nonce  uint64
+}
```

Comment:
Also consider moving this one below `TransientData` to be consistent with Rust and also in hierarchical terms.

```suggestion
// TransientDataLifespan is a structure representing the transient data lifespan.
type TransientDataLifespan struct {
	Origin abi.ActorID // The origin actor ID associated with the transient data.
	Nonce  uint64 // A unique nonce identifying the transaction.
}
```